packages/expo-apple-authentication/CHANGELOG.md

```diff
@@ -10,6 +10,9 @@
 
 ### 💡 Others
 
+- Extend the `AppleAuthenticationButton` component by `View`
+- Exclude `backgroundColor` and `borderRadius` from the `style` type
```

Comment:
- Exclude `backgroundColor` and `borderRadius` properties from the `AppleAuthenticationButton`'s `style` prop. These two are invalid for `AppleAuthenticationButton`, but TypeScript allowed the usage of them; instead use `buttonStyle` and `cornerRadius` props repsectively. ([#13567](https://github.com/expo/expo/pull/13567) by [@Simek](https://github.com/Simek))